app/components/modals/session-notify-modal.js

```diff
@@ -35,6 +36,12 @@ export default class SessionNotifyModal extends ModalBase {
     return mail;
   }
 
+  @computed('sessionId')
+  get speakerEmails() {
+    const session = this.store.peekRecord('session', this.sessionId, { backgroundReload: false });
```

Comment:
Why `backgroundReload: false`?

Reply:
Bcz I thought otherwise there will be reloading that will be visible but we can do it without specifying ```backgroundReload: false``` and can do it with default behaviour.

Reply:
Yes, please use this instead